public/js/languages.js

```diff
@@ -2,7 +2,7 @@ window.HLRDESK = window.HLRDESK || {};
 window.HLRDESK.init = window.HLRDESK.init || {};
 
 window.HLRDESK.init.languages = function initLanguages() {
-  var langSearch = document.getElementById('lang-search');
+  var langSearch = $('#lang-search').select2();
```

Comment:
I like how github recognizes this difference, but if you add a comma the whole line is "new"